app/views/tags/show/_organisations.html.erb

```diff
@@ -0,0 +1 @@
+<%= render "tags/show/multi_tag", service: "organisation",  values: edition.tags["organisations"].to_a %>
```

Comment:
I'm assuming this is the view shown on the document summary? For consistency with contents this should be in `app/views/documents/show/tags` and have a `_field` suffix in the name - same for similar.